packages/core/src/postProcess/PostProcess.ts

```
@@ -0,0 +1,132 @@
+   */
+  readonly isGlobal = true;
+
+  /**
+   * @internal
+   */
+  @ignoreClone
+  _effects: PostProcessEffect[] = [];
+
+  constructor(entity: Entity) {
+    super(entity);
+  }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)